### PR TITLE
finish adding ref value met_typical_tasks

### PR DESCRIPTION
**Pull request by Ilite-Kiriya**

### CODE REVIEW

File: src/utilities/utilities.js [hunk 1]
```diff
@@ -672,3 +672,80 @@ export function valid_range(range, [min, max]) {
...
+ * @property {number} Calisthenics - 3.5
+ * @property {number} Walking_4mph_6_4kmh - 3.8
+ * @property {number} Tennis - 3.8
+ * @property {number} Heavy_machine_work - 4.0
+ * @property {number} Handling_100lb_45_kg_bags - 4.0
+ * @property {number} Pick_and_shovel_work - 4.4
+ * @property {number} Basketball - 6.3
+ * @property {number} Wrestling - 7.8
+ * @example
+ * from pythermalcomfort.utilities import met_typical_tasks
```
Comment (szaldivar) [inline src/utilities/utilities.js:715]: the example is in python

**Review by szaldivar — CHANGES_REQUESTED** [review]

File: src/utilities/utilities.js (hunk 1 above)
Comment (Ilite-Kiriya) [reply src/utilities/utilities.js:715]: Should I delete this example?

Comment (szaldivar) [reply src/utilities/utilities.js:715]: write in JS

Comment (Ilite-Kiriya) [reply src/utilities/utilities.js:715]: ...🤣 Can't believe that I didn't realise I write the code in python

**Comment by Ilite-Kiriya** [comment]
Btw, I m not sure if
```import {met_typical_tasks} from "./src/utilities/utilities.js"; //The path to utilities.js```
is right hahh.

**Comment by szaldivar** [comment]
> Btw, I m not sure if `import {met_typical_tasks} from "./src/utilities/utilities.js"; //The path to utilities.js` is the most appropriate hahh.

maybe do 
```javascript
import { met_typical_tasks } from "jsthermalcomfort/utilities";
```

File: src/utilities/utilities.js [hunk 2]
```diff
@@ -672,3 +672,80 @@ export function valid_range(range, [min, max]) {
...
+ * @property {number} Calisthenics - 3.5
+ * @property {number} Walking_4mph_6_4kmh - 3.8
+ * @property {number} Tennis - 3.8
+ * @property {number} Heavy_machine_work - 4.0
+ * @property {number} Handling_100lb_45_kg_bags - 4.0
+ * @property {number} Pick_and_shovel_work - 4.4
+ * @property {number} Basketball - 6.3
+ * @property {number} Wrestling - 7.8
+ * @example
+ * import {met_typical_tasks} from "./src/utilities/utilities.js"; //The path to utilities.js
```
Comment (szaldivar) [inline src/utilities/utilities.js:715]:
```suggestion
 * import { met_typical_tasks } from "jsthermalcomfort/utilities"; //The path to utilities
```

**Review by szaldivar — CHANGES_REQUESTED** [review]

**Review by szaldivar — APPROVED** [review]

**Pull request merged**